### PR TITLE
Update Module readme template

### PR DESCRIPTION
**Pull request by sguequierre**
- Get rid of duplicative build and run sections
- Make substitution easier to understand by using API and Model triplet pairs as they appear in github (also was slightly inaccurate)
- Add note directing to modular resources docs 
- Update agilex-limo to match (can update real readme if this gets approved)

### CODE REVIEW

File: docs/registry/create/_index.md [hunk 1]
```diff
@@ -1099,11 +1099,14 @@ If you follow [the instructions to generate module scaffolding for a Python modu
...
 ````md
-# <INSERT NAME> modular resource
+# [<INSERT MODULE NAME> module](<INSERT LINK TO MODULE REPO>)
 
-This module implements the <INSERT API NAMESPACE> <INSERT API NAME> API in a <INSERT MODEL> model.
+This module implements the <INSERT API TRIPLET> API in a <INSERT MODEL> model.
 With this model, you can...
 
+> [!NOTE]
+> For more information, see [Modular Resources].
```
Comment (npentrel) [inline docs/registry/create/_index.md:1108]: missing link?

File: docs/registry/create/_index.md [hunk 2]
```diff
@@ -1171,20 +1170,19 @@ _Add troubleshooting notes here._
...
-# AgileX LIMO Driver
+# [`agilex-limo` module](https://app.viam.com/module/viam/agilex-limo)
 
 This module implements the base driver for the [AgileX LIMO](https://global.agilex.ai/education/4) base to be used with [`viam-server`](https://docs.viam.com/). This driver supports differential, ackermann, and omni directional steering modes over the serial port.
 
-## Build and run
-
-To use this module, follow these instructions to [add a module from the Viam Registry](https://docs.viam.com/registry/configure/#add-a-modular-resource-from-the-viam-registry) and select the `viam:base:agilex-limo` model from the [`agilex-limo` module](https://app.viam.com/module/viam/agilex-limo).
+> [!NOTE]
+> For more information, see [Modular Resources].
```
Comment (npentrel) [inline docs/registry/create/_index.md:1178]: missing link

**Review by npentrel — APPROVED** [review]
LGTM aside from missing links

File: docs/registry/create/_index.md (hunk 1 above)
Comment (sguequierre) [reply docs/registry/create/_index.md:1108]: oops

Comment (sguequierre) [inline docs/registry/create/_index.md:1108]:
```suggestion
> For more information, see [Modular Resources](https://docs.viam.com/registry/#modular-resources).
```

File: docs/registry/create/_index.md (hunk 2 above)
Comment (sguequierre) [inline docs/registry/create/_index.md:1178]:
```suggestion
> For more information, see [Modular Resources](https://docs.viam.com/registry/#modular-resources).
```

File: docs/registry/create/_index.md [hunk 3]
```diff
@@ -554,7 +554,7 @@ func (b *myBase) Close(ctx context.Context) error {
 {{< alert title="Note" color="note" >}}
 For an example featuring a sensor, see [MCP3004-8](https://github.com/mestcihazal/mcp3004-8-go).
 
-For additional examples use the [modular resources search](https://docs.viam.com/registry/#modular-resources) to search for examples of the model you are implementing, and click on the model's link to be able to browse its code.
+For additional examples use the [modular resources search](https://docs.viam.com/registry/#modular-resources) to examples of the model you are implementing, and click on the model's link to be able to browse its code.
```
Comment (sguequierre) [inline docs/registry/create/_index.md:557]:
```suggestion
For additional examples use the [modular resources search](https://docs.viam.com/registry/#modular-resources) to search for examples of the model you are implementing, and click on the model's link to be able to browse its code.
```

File: docs/registry/create/_index.md [hunk 4]
```diff
@@ -304,7 +304,7 @@ class MyBase(Base, Reconfigurable):
 {{< alert title="Note" color="note" >}}
 For an example featuring a sensor, see [MCP300x](https://github.com/viam-labs/mcp300x-adc-sensor).
 
-For additional examples use the [modular resources search](https://docs.viam.com/registry/#modular-resources) to search for examples of the model you are implementing, and click on the model's link to be able to browse its code.
+For additional examples use the [modular resources search](https://docs.viam.com/registry/#modular-resources) to examples of the model you are implementing, and click on the model's link to be able to browse its code.
```
Comment (sguequierre) [inline docs/registry/create/_index.md:307]:
```suggestion
For additional examples use the [modular resources search](https://docs.viam.com/registry/#modular-resources) to search for examples of the model you are implementing, and click on the model's link to be able to browse its code.
```

**Pull request merged**